.github/workflows/benchmark.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       MAIN-BENCH-16: ${{ steps.benchmark-main.outputs.BENCH_RESULT16 }}
```

Comment:
Remove the references to `MAIN-BENCH-12` and `PR-BENCH-12`

Reply:
Done 👍